coralproject/talk
# Websocket Regression

## What does this PR do?

Addresses websocket connection issues:

- Reverted upgrade to subscriptions-transport-ws
- Removed unnecessary websocket resets
- Improved websocket reconnection logic

## How do I test this PR?

1. Open Google Chrome (unfortunately, one of the few browsers with decent websocket debugging) to the admin page, ensure you're logged out.
2. Open the network tab in the developer tools, and filter by websockets by clicking "WS" at the filter bar.
3. Refresh the page, and notice no websocket connection is opened.
4. Login to the admin, notice the single websocket connection opened.
5. Logout of the admin, and notice the second websocket connection is opened, and the old one is closed.
